### ISSUE TITLE
Create Integration Tests

### ISSUE THREAD

**Issue by iantalarico** [GoogleCloudPlatform/google-cloud-visualstudio#926]
We should create integration tests to cover some of our major use cases.

**Comment by iantalarico** [comment]
@amanda-tarafa Do you want to take this?  Maybe start with a few simple tests that cover something easy to test?  @ILMTitan did the initial research on this and should be able to point you in the right direction.

**Comment by amanda-tarafa** [comment]
Yep, sounds good. I can start with tests for the Publish dialogs I've been changing, I know them by heart now.
